components/brave_webtorrent/stories/story.tsx

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+
+// Components
+import Viewer from '../extension/components/torrentViewer'
+
+// Styles
+require('../extension/styles/styles.css')
+
+// Storybook
+const fullPageStoryStyles: object = {
+  width: '-webkit-fill-available',
+  height: '-webkit-fill-available'
+}
+
+// Storybook helpers
+const FullPageStory = (storyFn: any) => (
+  <div style={fullPageStoryStyles}>{storyFn()}</div>
+)
+
+function consoleLog() {
+  console.log(`Clicked something`)
+}
+
+const sampleTorrent = {
+  name: 'Sample Torrent',
+  files: [{
+    name: 'file1.jpg',
+    length: 4000 // 4KB
+  }, {
+    name: 'file2.jpg',
+    length: 500
+  }],
+  serverURL: 'ros://myfile',
+  timeRemaining: 42,
+  downloaded: 2000,
+  uploaded: 8000,
+  downloadSpeed: 200,
+  uploadSpeed: 200,
+  progress: 5,
```

Comment:
@feross there aren't any units on these variables so didn't know if some of these were in the correct range, especially this 'progress' item. On second glance I'm wondering if this is meant to be between `0` and `1`?

Reply:
Progress should be between `0` and `1`. Most of these values are documented here: https://webtorrent.io/docs

Reply:
set to 0